chatcommands.lua

```diff
@@ -21,6 +21,16 @@ local create_channel = {
 	func = function(lname, param)
 		local lowner = lname
 
+		--check player privileges if plugin priv_cc is enabled
+		local name_priv2cc = minetest.settings:get("beerchat.name_priv_cc")
+		if minetest.settings:get_bool("beerchat.enable_priv_cc") then
```

Comment:
```suggestion
		if minetest.settings:get_bool("beerchat.enable_priv_cc") then
			local name_priv2cc = minetest.settings:get("beerchat.priv_cc.name")
```
To follow pattern used for other optional extensions: `beerchat.<plugin name>.<setting name>` and reduce checks to one if disabled, not important performance wise but I'd just like to skip unnecessary stuff whenever possible.